tests/test_util/test_util_typing.py

```diff
@@ -62,7 +62,7 @@ class MyEnum(Enum):
 MyInt = NewType('MyInt', int)
 
 
-class MyList(List[T]):
+class MyList(list[T]):
```

Comment:
This seems like a strange change, in isolation. The entire rest of the file is still using the capitalized `typing` imports. Why change only this _one_ class?

Reply:
That's a good question; I think the most truthful response is that it's a cautious approach, and that I was concerned that since the test expectations have `typing.List` within string literals, I didn't feel confident that they would be testing the same behaviour after a migration.

A naive evaluation of `type(list)` and `type(typing.List)` locally indicates that they remain non-identical.. so a more cautious approach could be to omit this change.

To attempt to extract a grander principle from all of the above, perhaps it's that automated code reformatting is fine, but that I sense some amount of risk in automatically rewriting test cases (because, ultimately, that could mean that a behaviour-change is obscured by what appears to be a stylistic modification).

Reply:
If the test-behaviour-change issue is a concern, and given commit `A` prior to the test suite change, and commit `B` subsequent to the change, then I suppose it could make sense to run both test suites from `A` and `B` against the application/library code from commit `B`, to assert that the changes in `B` did indeed not have any behavioural effect.

It feels like it may be insufficient to provide complete confirmation of no-behaviour-change, though.  It could simply be that the test coverage was insufficient to detect a change.

Reply:
Yes, this one needs to be like that. While `typing.List` is an alias of `list`, it's more a special alias. There are runtime differences between `typing.List` and `list` and those differences may matter for the tests (note that in the tests, I use `list` *and* `typing.List` to ensure that objects in the `builtins` and `typing` modules are correctly handled even though they are semantically equivalent).